### PR TITLE
Moving Aries Agent charts from Hyperledger Labs to Bevel

### PR DESCRIPTION
**Pull request by monar24**
Our team at superlogic proposed a hyperledger lab for our aries-agent charts. We were asked to raise a PR for contributing this part to Bevel. Previous conversation may be viewed here: https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/259

### CODE REVIEW

**Review by suvajit-sarkar — APPROVED** [review]
LGTM

File: platforms/hyperledger-aries/chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  group: 'vc-authn-oidc'
+  name: 'vc-authn-oidc'
+  role: 'agent'
+
+controller:
+  role: 'controller'
+  webhookBaseURL: ''
+
+deploy:
+  image:
```
Comment (sownak) [inline platforms/hyperledger-aries/chart/values.yaml:21]: can we use `image.repository` and `image.tag` format for the images

File: platforms/hyperledger-aries/chart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    name: 'ghcr.io/hyperledger/aries-cloudagent-python'
+    ## Modify aca-py agent version: 
+    tag: 'py3.9-0.9.0'
+  name:
+    prefix: 'aries-agent'
+    suffix: ''
+
+ingress:
```
Comment (sownak) [inline platforms/hyperledger-aries/chart/values.yaml:29]: Bevel calls ingress and proxy, so can this be moved to key called `global.proxy`

**Review by sownak — CHANGES_REQUESTED** [review]
How is multi-cloud supported?

**Comment by suvajit-sarkar** [comment]
Hi @monar24 i think it will make more sense to put the Aries helm charts in https://github.com/hyperledger/bevel-samples under identity rather having under Bevel DLT platforms. Or maybe we put under Indy platform as an identity solution. ?

**Comment by monar24** [comment]
> https://github.com/hyperledger/bevel-samples



> Hi @monar24 i think it will make more sense to put the Aries helm charts in https://github.com/hyperledger/bevel-samples under identity rather having under Bevel DLT platforms. Or maybe we put under Indy platform as an identity solution. ?

Thanks everyone for the feedback and time reviewing. 

I actually agree with moving it to bevel-samples. Aries agents are deployed in order to interact with an underlying ledger, the ledger being indy by default mostly. Since bevel itself is for deploying the actual networks, I agree that it may be better to move these agent charts there. 

I have raised that PR here: https://github.com/hyperledger/bevel-samples/pull/78

**Comment by suvajit-sarkar** [comment]
Closing this PR as we have moved to https://github.com/hyperledger/bevel-samples/pull/78

**Comment by swcurran** [comment]
Flagging @WadeBarnes about this contribution.

**Pull request closed without merging**